exercises/dominik_arominski/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include <fstream>
```

Comment:
It is a good idea to sort the includes alphabetically.

Reply:
Agreed.